ndbench-proxy.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
```

Comment:
@vinaykumarchella external users cannot make changes to the Wiki. @diegopacheco may be a Gist would help and I can port it to the Wiki.